src/vellum/workflows/events/tests/test_event.py

```diff
@@ -328,6 +328,7 @@ class MockWorkflow(BaseWorkflow[MockInputs, BaseState]):
                             "name": "default",
                         }
                     ],
+                    "mocked": None,
```

Comment:
We should clean this up and not have it appear on serialization: https://linear.app/vellum/issue/APO-146/skip-default-values-in-event-serialization-like-mocked-none